internal/riverinternaltest/riverdrivertest/riverdrivertest.go

```diff
@@ -2204,37 +2204,6 @@ func Exercise[TTx any](ctx context.Context, t *testing.T,
 
 	const leaderTTL = 10 * time.Second
 
-	t.Run("LeaderDeleteExpired", func(t *testing.T) {
```

Comment:
I just moved this set of tests down so it's in alphabetical order and easier to find.